BuilderDesignPattern/build.gradle

```diff
@@ -2,7 +2,7 @@ plugins {
     id 'java'
 }
 
-group 'org.example'
+group 'designPattern'

```

Comment:
Take a look at [this guide](https://docs.oracle.com/javase/tutorial/java/package/namingpkgs.html#:~:text=Companies%20use%20their%20reversed%20Internet%20domain%20name%20to%20begin%20their%20package%20names%E2%80%94for%20example%2C%20com.example.mypackage%20for%20a%20package%20named%20mypackage%20created%20by%20a%20programmer%20at%20example.com.) from Oracle which shows the general naming convention for group/package names

> Companies use their reversed Internet domain name to begin their package names—for example, com.example.mypackage for a package named mypackage created by a programmer at example.com.

 So the convention is to reverse a domain usually. Some people on GitHub use the GitHub domain followed by their username, so yours might be something like this:

`com.github.benrodriguez10`

Your package names in each of your `.java` files might become then something like `com.github.benrodriguez10.designpattern1`

If you choose to change the package names in this way, it's worth noting that Java expects to see package names honoured in the directory structure, e.g. if you had the above package name, the Java files would need to be in a directory like this:

```
BuilderDesignPattern/src/main/java/com/github/benrodriguez10/designpattern1
```